docker/Dockerfile.oldev

```diff
@@ -10,25 +10,22 @@ RUN touch /etc/authbind/byport/80 && chmod 777 /etc/authbind/byport/80
 
 WORKDIR /openlibrary
 
+# Set owner to be openlibrary user
+RUN chown -R openlibrary .
+
 # Setup nginx
 RUN ln -s /openlibrary/conf/nginx/sites-available/openlibrary.conf /etc/nginx/sites-available/ \
     && ln -s /etc/nginx/sites-available/openlibrary.conf /etc/nginx/sites-enabled/
 
-# SOLR
-RUN sed -i 's/8080/8983/g' /etc/tomcat7/server.xml \
-  && cp conf/solr/conf/schema.xml /etc/solr/conf/
-
 RUN mkdir -p /var/log/openlibrary /var/lib/openlibrary \
     && chown openlibrary:openlibrary /var/log/openlibrary /var/lib/openlibrary
 
 # TODO: remove vagrant bootstrap.sh and conf/init services
 
-# Update dev IA s3 endpoints to use container port 80
-# TODO: update openlibrary.yml directly once we have fully deprecated Vagrant
-RUN sed -i 's/127.0.0.1:8080/127.0.0.1/g' conf/openlibrary.yml
-
-# Replace Vagrant User with `openlibrary` in dev db dump
-RUN sed -i 's/vagrant/openlibrary/g' scripts/dev-instance/dev_db.pg_dump
+# TODO: Find safer way to update conf
```

Comment:
Yes, using `sed` was only meant to be temporary until we got rid of vagrant all together.

I've recently been using Docker for dev, and discovered a big downside to this sed approach is that when you mount your local dev directory in the container, you lose the substitutions that were made to the s3 endpoints, as they only exist in  the image :(

Reply:
Yeah, copying over stuff into the image is creating some weird development issues. I think we should consider switching to just a volume (no copying).